pyfar/classes/coordinates.py

```diff
@@ -1374,16 +1375,313 @@ def show(self, mask=None, **kwargs):
             pf.plot.scatter(self, **kwargs)
         else:
             mask = np.asarray(mask)
-            assert mask.shape == self.cshape, \
-                "'mask.shape' must be self.cshape"
-            colors = np.full(mask.shape, pf.plot.color('b'))
+            colors = np.full(self.cshape, pf.plot.color('b'))
             colors[mask] = pf.plot.color('r')
             pf.plot.scatter(self, c=colors.flatten(), **kwargs)
 
+    def find_nearest(self, find, k=1, distance_measure='euclidean'):
+        """
+        Find the k nearest coordinates points.
+
+        Parameters
+        ----------
+        find : pf.Coordinates
+            Coordinates to which the nearest neighbors are searched.
+        k : int, optional
+            Number of points to return. k must be > 0. The default is ``1``.
+        distance_measure : string, optional
+            ``'euclidean'``
+                Distance is determined by the euclidean distance i meters
+                between the points. This is default.
+            ``'spherical'``
+                Distance is determined by the great-circle distance in meters
+                between the points.
+            ``'angular'``
+                distance is determined by the angles in radiant.
+
+        Returns
+        -------
+        index : tuple of arrays
+            Indices of the neighbors. Arrays of shape (k, find.cshape) if k>1
+            else (find.cshape).
+        distance : numpy array of floats
+            Distance between the points, after the given ``distance_measure``.
+            It's of shape (k, find.cshape).
+
+        Notes
+        -----
+        This is a wrapper for ``scipy.spatial.cKDTree``.
+
+        Examples
+        --------
+
+        Find frontal point from a spherical coordinate system
+
+        .. plot::
+
+            >>> import pyfar as pf
+            >>> coords = pf.samplings.sph_lebedev(sh_order=10)
+            >>> find = pf.Coordinates(1, 0, 0)
+            >>> index, distance = coords.find_nearest(find)
```

Comment:
```suggestion
            >>> to_find = pf.Coordinates(1, 0, 0)
            >>> index, distance = coords.find_nearest(to_find)
```
I feel like this makes it a bit clearer. But it's not that important.